dashboard-ui/src/components/ActionsCompare/scenarioDetails.jsx

```diff
@@ -0,0 +1,104 @@
+import React from 'react';
+import { Accordion, ListGroup } from 'react-bootstrap';
+import { nameMappings } from './decisionList';
+import { Query } from 'react-apollo';
+import gql from 'graphql-tag';
+
+const getScenarioQuery = gql`
+    query getScenario($scenarioId: ID) {
+        getScenario(scenarioId: $scenarioId) 
+    }`;
+class ScenarioDetails extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            scenarioId: this.props.selectedScenario
+        };
+    }
+
+    renderPatientInfo(patient) {
+        return (
+            <div className="row">
+                <div className="col-md-6">
+                    <h3>Demographic Information</h3>
+                    <ListGroup variant="flush">
+                        <ListGroup.Item>Age: {patient.demographics.age}</ListGroup.Item>
+                        <ListGroup.Item>Sex: {patient.demographics.sex}</ListGroup.Item>
+                        <ListGroup.Item>Rank: {patient.demographics.rank}</ListGroup.Item>
+                    </ListGroup>
+                </div>
+                <div className="col-md-6">
+                    <h3>Injuries</h3>
+                    <ListGroup variant="flush">
+                        {patient.injuries.map((injury, index) => (
+                            <ListGroup.Item key={index}>
+                                <strong>Name:</strong> {injury.name}<br />
+                                <strong>Location:</strong> {injury.location}<br />
+                                <strong>Severity:</strong> {injury.severity}
+                            </ListGroup.Item>
+                        ))}
+                    </ListGroup>
+                </div>
+            </div>
+        );
+    }
+    
+    
+
+    componentDidUpdate(prevProps) {
+        if (this.props.selectedScenario !== prevProps.selectedScenario) {
+            this.setState({ scenarioId: this.props.selectedScenario })
+        }
+    }
+
+    render = () => {
+        return (
+            <Accordion defaultActiveKey="0">
```

Comment:
It would be nice to remove the Accordion from the top level of the Scenario Details, but keep the Accordion for the patients.   I think having a few details always visible would be good for the page.